asserts/systestkeys/trusted.go

```diff
@@ -230,14 +287,49 @@ iJkSeYilqoKo3AJhnICRiqvAca2TGJoiJUryEgZ8jbTOElfaF2p+y0xvXGlWbKZm1gzGyvFM5fV5
 hJTlp/am+2uVn6U8wPACir4PrbuXYo7L4MIXww2OEO0ruBIaLARbc5IutSWmw6AEYQUxtsa9bdHV
 Zin7LGbEj6lZm8GycWQwh4B6Vnt6dJRIyPc/9G7uM8Ds/2Wa7+yAxhiPqm8DwlbOYh1npw4X4TLD
 IMGnTv5N3zllI+Xz4rqJzNTzEbvOIcrqWxCedQe79A==
+`
+
+	TestRepairKeyID = "vOg8BXaG8hNaHq-Hccyq4Pi4x_Ze11bWDn2gGSo2Z4s_hIcJZEe04ujqVPmuKt08"
+
+	encodedTestRepairRootAccountKey = `type: account-key
+authority-id: testrootorg
+public-key-sha3-384: vOg8BXaG8hNaHq-Hccyq4Pi4x_Ze11bWDn2gGSo2Z4s_hIcJZEe04ujqVPmuKt08
+account-id: testrootrepair
```

Comment:
this should be testrootorg as well,  things might work with this because there's a missing check in runner.go that we should add though it's not super material in practice